docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -18,85 +18,43 @@ We'll cover the following topics:
 
 ## Installation
 
-The controller package provides a service unit and, if bootstrapping is enabled, will generate a configuration based on the bootstrapping answer file. The `openziti` package provides the `ziti` CLI and is installed automatically as a dependency. Run the install script for RPM and Debian distributions or manually configure the package repo and install package `openziti-controller`.
+The controller package provides a systemd service unit and bootstrapping script.
 
-Download and run the install script.
+Run the install script.
 
 ```text
-wget https://get.openziti.io/install.bash
+curl -sS https://get.openziti.io/install.bash | sudo bash -s openziti-controller
 ```
 
-Install interactively so that you will have an opportunity to answer questions about generating a configuration.
-
-```text
-sudo bash ./install.bash openziti-controller
-```
+Or, manually configure the package repo and install `openziti-controller`.
 
 <LinuxPackageRepo />
 
+The `openziti` package provides the `ziti` CLI and is installed as a dependency.
+
 ## Configuration
 
-You must provide or generate a configuration. Configuration consists of a PKI, a config YAML file, and a database.
+You must generate, migrate, or craft a configuration. Configuration consists of a PKI, a config YAML file, and a database.

```

Comment:
in practicality, migrating or creating (the word 'craft' feels odd for some reason) are the same thing to me. Can we just eliminate this and go right "generate a configuration" and leave some kind of footnote/link to how to migrate/edit the configuration after it's generated? i don't think it deserves a 'heading' in this case